pkg/subctl/cmd/gather.go

```diff
@@ -148,6 +170,8 @@ func gatherConnectivity(dataType string, info gather.Info) bool {
 		gather.RouteAgentPodLogs(info)
 		gather.GlobalnetPodLogs(info)
 		gather.NetworkPluginSyncerPodLogs(info)
+		gather.CNIResources(info)
+		gather.CableDriverResources(info)
```

Comment:
Shouldn't these be under the `Resources` type?

Reply:
Yeah, I agree, better to make them resource types.